src/lineage/entrypoint.py

```diff
@@ -90,8 +90,9 @@ def load_template(github_workspace_dir, default_filename, custom_filename=None):
     else:
         logging.info("Loading default template: %s", default_filename)
         template_data = (
-            pkg_resources.resource_string("lineage", f"templates/{default_filename}")
-            .decode("utf-8")
+            files(f"{__package__}.templates")
+            .joinpath(default_filename)
```

Comment:
This puts this in line with the changes in https://github.com/cisagov/skeleton-python-library/pull/150.

```suggestion
            files(__package__)
            .joinpath("templates", default_filename)
```

Reply:
Please see commit 13945f9.